File: test/add_test_data.js
const db = require('./config/db');

async function addTestData() {
  try {
    // Add some test customers
    const customers = [
      { username: 'John Doe', email: 'john@example.com', contacts: 123456789, service: 'Web Development', details: 'Looking for a website redesign' },
      { username: 'Jane Smith', email: 'jane@example.com', contacts: 987654321, service: 'Mobile App', details: 'Need a mobile app for my business' },
      { username: 'Bob Johnson', email: 'bob@example.com', contacts: 555555, service: 'Web Development', details: 'E-commerce website needed' },
      { username: 'Alice Brown', email: 'alice@example.com', contacts: 1111, service: 'Consulting', details: 'Business process optimization' },
      { username: 'Charlie Wilson', email: 'charlie@example.com', contacts: 22222222, service: 'Mobile App', details: 'iOS app development' },
      { username: 'Diana Davis', email: 'diana@example.com', contacts: 3333333, service: 'Web Development', details: 'Corporate website design' },
      { username: 'Eve Miller', email: 'eve@example.com', contacts: 4444444, service: 'Consulting', details: 'Market research and analysis' }
    ];

    for (const customer of customers) {
      const sql = `INSERT INTO customers(username, user_id, email, contacts, service, details) VALUES (?, ?, ?, ?, ?, ?)`;
      const [result] = await db.execute(sql, [customer.username, 1, customer.email, customer.contacts, customer.service, customer.details]);
      console.log(`Added customer: ${customer.username}`);
    }

    console.log('Test data added successfully!');
    process.exit(0);
  } catch (error) {
    console.error('Error adding test data:', error);
    process.exit(1);
  }
}

addTestData();

File: trainer/add_test_data.py
#!/usr/bin/env python3
"""
Script to add test data to the CRM database.
"""

import mysql.connector
import os
from dotenv import load_dotenv

# Load environment variables
load_dotenv()

def get_db_connection():
    """Create a database connection using environment variables"""
    return mysql.connector.connect(
        host=os.getenv('DB_HOST', 'localhost'),
        user=os.getenv('DB_USER', 'root'),
        password=os.getenv('DB_PASSWORD', 'A002#tz1'),
        database=os.getenv('DB_NAME', 'crm'),
        port=int(os.getenv('DB_PORT', 3306))
    )

def add_test_data():
    """Add test customer data to the database"""
    try:
        # Create database connection
        connection = get_db_connection()
        cursor = connection.cursor()
        
        # Test data for customers
        customers = [
            ('John Doe', 1, 'john@example.com', '123456789', 'Web Development', 'Looking for a website redesign'),
            ('Jane Smith', 1, 'jane@example.com', '987654321', 'Mobile App', 'Need a mobile app for my business'),
            ('Bob Johnson', 1, 'bob@example.com', '555555', 'Web Development', 'E-commerce website needed'),
            ('Alice Brown', 1, 'alice@example.com', '1111', 'Consulting', 'Business process optimization'),
            ('Charlie Wilson', 1, 'charlie@example.com', '22222222', 'Mobile App', 'iOS app development'),
            ('Diana Davis', 1, 'diana@example.com', '333333', 'Web Development', 'Corporate website design'),
            ('Eve Miller', 1, 'eve@example.com', '4444444', 'Consulting', 'Market research and analysis')
        ]
        
        # Insert test data
        sql = "INSERT INTO customers(username, user_id, email, contacts, service, details) VALUES (%s, %s, %s, %s, %s, %s)"
        
        for customer in customers:
            cursor.execute(sql, customer)
            print(f"Added customer: {customer[0]}")
        
        # Commit changes
        connection.commit()
        print("Test data added successfully!")
        
        # Close connection
        cursor.close()
        connection.close()
        
    except Exception as e:
        print(f"Error adding test data: {e}")
        return False
    
    return True

if __name__ == "__main__":
    success = add_test_data()
    exit(0 if success else 1)